util/src/MetricsServer.cpp

```diff
@@ -137,7 +137,7 @@ void Server::sendError(sockaddr_in* cliaddr, socklen_t addrlen) {
   memcpy(buf_ + sizeof(Header), msg, msglen);
   auto len = sendto(sock_, buf_, msglen + sizeof(Header), 0, (const struct sockaddr*)cliaddr, addrlen);
   if (len < 0) {
-    LOG_ERROR(logger_, "Failed to send error msg: " << concordUtils::errnoString(errno));
```

Comment:
I would consider leaving this as an error due to the following explanation of the sendto system call which should be also applicable to sendto.
"No indication of failure to deliver is implicit in a send(). Locally detected errors are indicated by a return value of -1."
In other words an error would most likely be a local failure and won't reveal a byzantine peer that didn't deliver the message. Especially since this send is over UDP made by the metrics server.

Reply:
Agreed. 
The only way this is not an error is during the server shutdown.
However, this may happen even in the regular case. leaving it in error level won't pass the ERROR checks (unless you are going to change it to detect only FATAL logs)
WDYT?

Reply:
We've already made the change to log FATAL only, I guess errors will need to be investigated from time to time.